draft-ietf-rats-uccs.md

```diff
@@ -305,39 +305,42 @@ FCFS space, with the present document as the specification reference.
 
 The security considerations of {{-cbor}} apply.
 The security considerations of {{-cwt}} need to be applied analogously,
-replacing the role of COSE with that of the Secured Channel.
+replacing the function of COSE with that of the secured channel.
 
-{{secchan}} discusses security considerations for Secure Channels, in which
+{{secchan}} discusses security considerations for secure channels, in which
 UCCS might be used.
 This document provides the CBOR tag definition for UCCS and a discussion
 on security consideration for the use of UCCS in
-Remote ATtestation procedureS (RATS).  Uses of UCCS outside the scope of
-RATS are not covered by this document.  The UCCS specification - and the
-use of the UCCS CBOR tag, correspondingly - is not intended for use in a
+Remote Attestation Procedures (RATS).  Uses of UCCS outside the scope of
+RATS are not covered by this document.  The UCCS specification -- and the
+use of the UCCS CBOR tag, correspondingly -- is not intended for use in a
 scope where a scope-specific security consideration discussion has not
 been conducted, vetted and approved for that use.
 
 ## General Considerations
 
-Implementations of Secure Channels are often separate from the application
+Implementations of secure channels are often separate from the application
 logic that has security requirements on them.  Similar security
 considerations to those described in {{-cose-new-struct}} for obtaining the
 required levels of assurance include:
 
 * Implementations need to provide sufficient protection for private or
-  secret key material used to establish or protect the Secure Channel.
+  secret key material used to establish or protect the secure channel.
 * Using a key for more than one algorithm can leak information about the
   key and is not recommended.
-* An algorithm used to establish or protect the Secure Channel may have
+* An algorithm used to establish or protect the secure channel may have
   limits on the number of times that a key can be used without leaking
   information about the key.
+* Evidence in a UCCS conveyed in a secure channel generally cannot be
```

Comment:
Except if the same key would've been used in both cases -- which seems to be the assumption here?